webapp/src/routes/home/instruction/instruction.component.js

```diff
@@ -0,0 +1,52 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { injectIntl, FormattedMessage } from 'react-intl';
+import { compose } from 'ramda';
+import homeIcon from '../../../images/game/ui/homescreen_icon.png';
+import upgradeIcon from '../../../images/game/ui/upgradescreen_icon.png';
+
+import { Container, Box } from './instruction.styles';
+import { Button } from './../../../theme/typography';
+import { Icon } from './components/icon';
+import messages from './instruction.messages';
+
+export const INSTRUCTION_READED = 'instruction_readed';
+
+export class InstructionComponent extends PureComponent {
+  static propTypes = {
+    hideInstruction: PropTypes.func.isRequired,
+  };
+  handleClose = () => {
+    localStorage.setItem(INSTRUCTION_READED, true);
+    this.props.hideInstruction();
+  };
+
+  render() {
+    return (
+      <Container>
+        <Box>
+          <p>Welcome to Farmtension, a farm-themed PWA game made at Apptension’s DevTalks.</p>
```

Comment:
jak już wszystko trzymamy w plikach z tłumaczeniami to tutaj też nie wrzucajmy tekstów bezpośrednio do kodu. Można użyć `<FormattedHtmlMessage />` dla ułatwienia i dodatkowo przekazać ikony przez zmienne (w razie problemów podejdę pokazać jak to zrobić)